DocumentFormat.OpenXml/src/Framework/OpenXmlPackage.cs

```diff
@@ -2079,7 +2079,9 @@ protected static Package FromFlatOpcDocumentCore(XDocument document, Package pac
     /// <summary>
     /// Specifies event handlers that will handle OpenXmlPackage validation events and the OpenXmlPackageValidationEventArgs. 
     /// </summary>
-    [Obsolete(ObsoleteAttributeMessages.ObsoleteV1ValidationFunctionality, false)]
+    // Building on Travis CI failed, saying that ObsoleteAttributeMessages does not contain a definition
+    // for 'ObsoleteV1ValidationFunctionality'. Thus, we've replaced the member with its value.
+    [Obsolete("This functionality is obsolete and will be removed from future version release. Please see OpenXmlValidator class for supported validation functionality.", false)]
```

Comment:
This is indeed really weird, as it works fine a few lines below. I'll file an mcs bug.